benchmarks_gui/CMakeLists.txt

```diff
@@ -41,13 +51,17 @@ include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 link_directories(${Boost_LIBRARY_DIRS})
 link_directories(${catkin_LIBRARY_DIRS})
 
-qt4_wrap_cpp(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
-qt4_wrap_ui(UIC_FILES
-  src/ui/main_window.ui
+set(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
```

Comment:
I wasn't complaining about the content. Only asking to rename MOC_HEADERS to headers (as elsewhere used).
Exactly for your reason b) to have the headers listed in QtCreator ;-)